.eslintrc

```diff
@@ -27,12 +27,12 @@
   "rules": {
     /* || Eslint main rules || */
     "camelcase": [2, { "properties": "always" }], // Override airbnb style.
-    "func-names": 2,
+    // "func-names": 2,
```

Comment:
This rule as well.  Why are we changing it?

Reply:
This rule was commented out here because it was added to the audit on line 119. We will activate it once all errors are fixed.

<img width="828" alt="Screen Shot 2020-05-22 at 9 46 39 AM" src="https://user-images.githubusercontent.com/55560129/82674605-91a35900-9c11-11ea-9b45-6f0612fb4fa3.png">